subsys/logging/log_backend_rtt.c

```diff
@@ -37,15 +37,20 @@
 
 #endif /* CONFIG_LOG_BACKEND_RTT_BUFFER > 0 */
 
-static const char *drop_msg = DROP_MSG;
 static u8_t rtt_buf[RTT_BUFFER_SIZE];
 static u8_t line_buf[CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE + DROP_MSG_LEN];
 static u8_t *line_pos;
 static u8_t char_buf;
 static int drop_cnt;
-static int drop_warn;
 static int panic_mode;
 
+#if CONFIG_LOG_BACKEND_RTT_MODE_DROP
```

Comment:
it's not needed. If variables are not used because of `if(0)` condition at compile time, variables will be removed and there will be no compilation warning.